submissions/YanaGushlevskaya/html-css-popup/gulpfile.js

```diff
@@ -0,0 +1,82 @@
+const gulp = require('gulp');
```

Comment:
Task does not imply use JavaScript

Reply:
@Valkirin `no JavaScript, only HTML/CSS, use of preprocessors is allowed`
Read about [Gulp](https://css-tricks.com/gulp-for-beginners/).
gulpfile.js-  configuration file for gulp. It helps with minification, adding prefix automatically etc.

@YanaGushlevskaya don't pay attention to this comment :smile: